vagrant-tramp.el

```diff
@@ -1,9 +1,36 @@
 ;;; vagrant-tramp.el --- Vagrant method for TRAMP
 
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict.. I added you as a collaborator to the repo, feel free to fix and merge this.  Thanks!